### ISSUE TITLE
Shouldn't show my earlier review comments while reviewing later commits.

### ISSUE THREAD

**Issue by Ms2ger** [Reviewable/Reviewable#136]
I'm reviewing <https://reviewable.io/reviews/servo/servo/6779> right now in "Review each commit separately" mode (thanks for that, btw!), and made quite a few comments on the first commit. Now all these comments show up on each following commit, and I have to manually figure out if there's any changes I need to review in the files that have them.

**Comment by pkaminski** [comment]
Yep, I realized this early on and plan to address it via #94 by letting you hide all comments (while still being able to create new ones).  I'm busy with the help / onboarding functionality right now, but after that this issue is pretty high on my to-do list.
